dynamicRooms/README.md

```diff
@@ -0,0 +1,21 @@
+# RSCBot: combineRooms
+
+The `dynamicRooms` cog is primarily responsible for enabling automatic voice channel management based server usage. This involves dynamically creating and deleting voice channels under a set category.
+
+## Installation
+
+The `dynamicRooms` cog is independant from all other cogs. No prerequisites are required.
+
+```
+<p>cog install RSCBot combineRooms
```

Comment:
I assume this is meant to be dynamicRooms here instead of combineRooms

Reply:
yes

Reply:
done